mode/tools/SDKUpdater/build.gradle

```diff
@@ -12,21 +12,57 @@ dependencies {
 
     implementationCopy group: "com.android.tools", name: "sdklib", version: "${toolsLibVersion}"
     implementationCopy group: "com.android.tools", name: "repository", version: "${toolsLibVersion}"
+    implementationCopy group: "com.android.tools", name: "common", version: "${toolsLibVersion}"
 }
 
 // This task copies the gradle tooling jar into the mode folder
 task copyToLib(type: Copy) {
   from configurations.implementationCopy.files {
-    include '**/annotations-*'
-    include '**/common-*'
-    include '**/dvlib-*'
-    include '**/gson-*'
-    include '**/guava-*'
-    include '**/httpcore-*'
-    include '**/kotlin-stdlib-*'
-    include '**/layoutlib-api-*'
-    include '**/repository-*'
-    include '**/sdklib-*'
+        include '**/*.jar'
+        exclude '**/annotations-*'
```

Comment:
just take care of wat you are removing from the final fat jar. And to test it you can try android-mode as external library in an android project.

Make sure that no dependency get excluded from the final fat jar.

Thanks !

Reply:
sure Aditya, I started by removing the jars, and whenever I got the class not found the issue, I added those specific class jars. I made sure both the Query and Download Swing Worker tasks went well. But I will do some more testing as you suggested to make sure no necessary dependency gets excluded.

Thanks for highlighting the importance

Reply:
> sure Aditya, I started by removing the jars, and whenever I got the class not found the issue, I added those specific class jars. I made sure both the Query and Download Swing Worker tasks went well. But I will do some more testing as you suggested to make sure no necessary dependency gets excluded.
> 
> Thanks for highlighting the importance

hey @rupeshkumar22 any update on this ?

Thanks !

Reply:
Hi @ranaaditya 
I am done with the testing and the updater faces no issues in fetching/installing the SDK data and reflecting back on the updater's UI. The included libraries in build.gradle are the only necessary and sufficient libraries to do the required operation of fetch or install.